save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -507,6 +511,32 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
 
     @Suppress("LongMethod", "TOO_LONG_FUNCTION", "MAGIC_NUMBER")
     private fun ChildrenBuilder.renderOrganizationMenuBar() {
+        modalAvatarBuilder(
+            isOpen = state.isAvatarWindowOpen,
+            title = "Change organization's avatar",
+            onCloseButtonPressed = {
+                setState {
+                    isAvatarWindowOpen = false
+                }
+            }
+        ) {
+            div {
```

Comment:
this part looks a common for `OrganizationView` and `UserSettingsView` -- maybe move it to modalAvatarBuilder?

Reply:
Yes. Moved it